weather_dl_v2/downloader_kubernetes/downloader_config.py

```diff
@@ -0,0 +1,38 @@
+import dataclasses
+import typing as t
+import json
+
+Values = t.Union[t.List["Values"], t.Dict[str, "Values"], bool, int, float, str]  # pytype: disable=not-supported-yet
+
+
+@dataclasses.dataclass
+class DownloaderConfig:
+    manifest_collection: str = ""
+    kwargs: t.Optional[t.Dict[str, Values]] = dataclasses.field(default_factory=dict)
+
+    @classmethod
+    def from_dict(cls, config: t.Dict):
+        config_instance = cls()
+
+        for key, value in config.items():
+            if hasattr(config_instance, key):
+                setattr(config_instance, key, value)
+            else:
+                config_instance.kwargs[key] = value
+
+        return config_instance
+
+
+downloader_config = None
+
+
+def get_config():
+    global downloader_config
+    downloader_config_json = "downloader_config.json"
+
+    if downloader_config is None:
+        with open(downloader_config_json) as file:
+            firestore_dict = json.load(file)
```

Comment:
Let's not name this as "firestore_dict". Put some more meaningful name to it.

Reply:
updated in [3c22d20](https://github.com/google/weather-tools/pull/384/commits/3c22d20406ff53ccd631f7b839ec789bd4010130)